chains/stridetestnet/metadata.yaml

```diff
@@ -0,0 +1,35 @@
+# yaml-language-server: $schema=../schema.json
+bech32Prefix: stride
+blockExplorers:
+  - apiUrl: https://testnet.ping.pub/stride
+    family: other
+    name: Mintscan
+    url: https://testnet.ping.pub/stride
+blocks:
+  confirmations: 1
+  estimateBlockTime: 5
+  reorgPeriod: 1
+chainId: stride-internal-1
+deployer:
+  name: Stride Labs
+  url: https://www.stride.zone
+displayName: Stride
```

Comment:
Should this be Stride Testnet?

Reply:
resolved [here](https://github.com/hyperlane-xyz/hyperlane-registry/pull/141/commits/dd7a361add9cee0f80fdcdbf5a28a21a39e54149)